benchmarks/tpch/justfile

```diff
@@ -0,0 +1,52 @@
+set fallback
+VENV := ".venv"
+VENV_BIN := ".venv/bin"
+
+
+@venv:
+	@python3 -m pip install virtualenv > /dev/null
```

Comment:
> other justfiles use a couple of spaces rather than an apparent tab.

my editor did the tabs. no standard fmtter for justfiles, so it didnt auto fix. 

> also here and 12,13, redirecting stdout to /dev/null just seems like it would be confusing? can we use a quiet mode instead?

the quiet mode doesn't seem to work here & dumps out a wall of text, so like in `py-glaredb/justfile` we just redirect to /dev/null to make it actually quiet.